### ISSUE TITLE
How to undo swipe to dismiss

### ISSUE THREAD

**Issue by JeffreyLeeSpokeo** [davideas/FlexibleAdapter#290]
So I was able to implement swipe to dismiss, but I've checked the documentation and I don't see a way to undo that. I am currently implementing it with a front view and a rear view.

https://github.com/davideas/FlexibleAdapter/wiki/5.x-%7C-Drag&Drop-and-Swipe

If I comment out my code that removes the item from the adapter and refreshes it, I stay on the rear view after swiping away the front view. I can swipe my front view back in. What I want is programmatically swipe my front view back in. Is there a way to do that?

**Comment by davideas** [comment]
@JeffreyLeeSpokeo, thanks for your message, I was trying hardly to achieve the automatic back swipe via code, without success, and also half swipe, but I gave up, when I saw that the android class was not for this purpose: only swipe-to-delete.
Months ago I added this limitation in the library. So, currently it is not possible, at least i couldn't do it.

if somebody wants to try, he is free to add this feature :+1:
